clarifai/client/dataset.py

```diff
@@ -37,13 +37,15 @@ def __init__(self,
                url: str = "",
                dataset_id: str = "",
                base_url: str = "https://api.clarifai.com",
+               pat: str = "",
```

Comment:
Perhaps a more pythonic way would be to default these to `None` instead of `""`. That way in checks you can do `if not param` as well, for example.